dashboard/test/ui/features/hamburger.feature

```diff
@@ -0,0 +1,177 @@
+@no_mobile
+Feature: Hamburger dropdown
+
+  Scenario: Signed out user in English should not see hamburger on desktop
+    Given I am on "http://code.org/"
+    And I dismiss the language selector
+    Then I wait until element ".header_button" is visible
+    Then I wait until element "#hamburger-icon" is not visible
+
+  Scenario: Student viewing hamburger dropdown in English on desktop on Code.org
+    Given I am on "http://code.org/"
+    And I dismiss the language selector
+    And I create a student named "Sally Student"
+    Then I wait to see "#hamburger-icon"
+    And I click selector "#hamburger-icon"
+    Then I wait to see "#hamburger-contents"
+    And I see "#report-bug"
+    And I see "#support"
+    And I see ".divider#before-pegasus"
+    And I see "#learn"
+    And I see "#stats"
+    And I see "#help-us"
+    And I see "#about_entries"
+    And I see "#educate_entries"
+
+  Scenario: Teacher viewing hamburger dropdown (with expanded options) in English on desktop on Code.org
+    Given I am on "http://code.org/"
+    And I dismiss the language selector
+    And I create a teacher named "Tessa Teacher"
+    Then I wait to see "#hamburger-icon"
+    And I click selector "#hamburger-icon"
+    Then I wait to see "#hamburger-contents"
+    And I see "#report-bug"
+    And I see "#support"
+    And I see "#teacher-community"
+    And I see ".divider#before-pegasus"
+    And I see "#learn"
+    And I see "#stats"
+    And I see "#help-us"
+    And I see "#about_entries"
+    And I click selector "#about_entries"
+    And I wait to see "#about-us"
+    And I see "#educate_entries"
+    And I click selector "#educate_entries"
+    And I wait to see "#educate-overview"
+
+  Scenario: Signed out user viewing hamburger dropdown in Spanish on desktop on Code.org
+    Given I am on "http://code.org/lang/es"
+    Then I wait to see "#hamburger-icon"
+    And I click selector "#hamburger-icon"
+    Then I wait to see "#hamburger-contents"
+    And I see "#report-bug"
+    And I see "#support"
+    Then I wait until element "#teacher-community" is not visible
```

Comment:
Do we need to wait for all of these to go away?  Or will they all be gone once `#support` is visible.